subcommittee/tooling/meetings/22-November-2024/minutes.md

```diff
@@ -0,0 +1,76 @@
+# Meeting 22.11.2024
+
+## Attendees
+- Tony Aiello (AdaCore)
+- Arnaud Fontaine (Airbus)
+- Johann Hemmann (Ferrous Systems)
+- Bogdan Genis (OxidOS)
+- Felix Gilcher (Ferrous Systems)
+- Julius Gustavson (Volvo Cars)
+- Tiago Manczak
+- Jordan McQueen (Woven by Toyota)
+- Orson Pessin (OxidOS)
+- Sasha Pourcelot (Trust in Soft)
+- Alexandru Radovici (Rust Foundation)
```

Comment:
I assume you meant to put Rust Foundation here as you were rep your position as a member or on the Board, but just wanted to make sure you didn't really mean to put OxidOs?

Reply:
What I meant is that I am just the moderator here, with the interest of pushing the work forward, rather than representing the interest of OxidOS. This is why I wrote Rust Foundation. Maybe I should write this differently?

Reply:
Right. Got it. Rust Foundation may be fine in that capacity, but could also probably just put "Moderator / OxidOS" or something too.

Reply:
To avoid confusion, I will just write Moderator.